src/Data/Array/Accelerate/Linear/Matrix.hs

```diff
@@ -50,14 +50,12 @@ import Control.Applicative
 import Prelude                                  as P
 
 
+
 infixl 7 !*!
 -- | Matrix product. This can compute any combination of sparse and dense multiplication.
 --
--- >>> V2 (V3 1 2 3) (V3 4 5 6) !*! V3 (V2 1 2) (V2 3 4) (V2 4 5)
--- V2 (V2 19 25) (V2 43 58)
---
--- >>> V2 (fromList [(1,2)]) (fromList [(2,3)]) !*! fromList [(1,V3 0 0 1), (2, V3 0 0 5)]
--- V2 (V3 0 0 2) (V3 0 0 15)
```

Comment:
I opted to delete the second example, since it relied on using an `IntMap`, which I don't believe is an instance of `Lift`/`Unlift`.